File: CS/WindowsFormsApplication1/Form1.cs
using System;
using System.Windows.Forms;
using DevExpress.XtraReports.UI;
using DevExpress.XtraReports.Expressions;
// ...

namespace WindowsFormsApplication1 {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e) {
            // Create a new report instance.
            XtraReport1 report = new XtraReport1();

            // Invoke the End-User Report Designer form.
            ReportDesignTool designTool = new ReportDesignTool(report);
            designTool.ShowDesignerDialog();
        }

    }
}

File: VB/WindowsFormsApplication1/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Windows.Forms
Imports DevExpress.XtraReports.UI
' ...

Namespace WindowsFormsApplication1
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

        Private Sub button1_Click(ByVal sender As Object, _
                                  ByVal e As EventArgs) Handles button1.Click
            ' Create a new report instance.
            Dim report As New XtraReport1()

            ' Invoke the End-User Designer form for the report.
            Dim designTool As New ReportDesignTool(report)
            designTool.ShowDesignerDialog()
        End Sub

	End Class
End Namespace
